.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 3 * * 6'
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8]
+          services: [lowest, release, devel]
+          include:
+          - services: lowest
+            REQUIREMENTS_LEVEL: "min"
+            EXTRAS: "all"
+          - services: release
+            REQUIREMENTS_LEVEL: "pypi"
+            EXTRAS: "all"
+          - services: devel
```

Comment:
We do not need `devel` anymore see https://github.com/inveniosoftware/invenio-records/pull/236#discussion_r533411854

Reply:
Yeah, this was done before.. removing it 👍